Formula/d/doppler-cli.rb

```diff
@@ -0,0 +1,27 @@
+class DopplerCli < Formula
```

Comment:
```suggestion
class Doppler < Formula
```

their tap is also using Doppler for the formula.https://github.com/DopplerHQ/homebrew-doppler/blob/master/Formula/doppler.rb